textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/join/JoinTest.java

```diff
@@ -1,2015 +0,0 @@
-    private Analyzer analyzer;
-    private Join join;
-    List<ITuple> bookTuple1;
-    List<ITuple> bookTuple2;
-    List<Attribute> attributeList;
-    List<Attribute> modifiedAttributeList;
-    private int maxVal = Integer.MAX_VALUE;
-
-    // This method sets up some stuff before beginning each test.
-    @Before
-    public void setup() throws Exception {
-        analyzer = new StandardAnalyzer();
-
-        final Attribute idAttr = new Attribute("id", FieldType.INTEGER);
-        final Attribute authorAttr = new Attribute("author", FieldType.STRING);
-        final Attribute titleAttr = new Attribute("title", FieldType.STRING);
-        final Attribute pagesAttr = new Attribute("numberOfPages", FieldType.INTEGER);
-        final Attribute reviewAttr = new Attribute("reviewOfBook", FieldType.TEXT);
-
-        final Attribute[] bookAttr = { idAttr, authorAttr, titleAttr, pagesAttr, reviewAttr };
-        final Schema bookSchema = new Schema(bookAttr);
-
-        IField[] book1 = { new IntegerField(52), new StringField("Mary Roach"),
-                new StringField("Grunt: The Curious Science of Humans at War"), new IntegerField(288),
-                new TextField("It takes a special kind " + "of writer to make topics ranging from death to our "
-                        + "gastrointestinal tract interesting (sometimes "
-                        + "hilariously so), and pop science writer Mary Roach is " + "always up to the task.") };
-
-        IField[] book2 = { new IntegerField(62), new StringField("Siddhartha Mukherjee"),
-                new StringField("The Gene: An Intimate History"), new IntegerField(608),
-                new TextField("In 2010, Siddhartha Mukherjee was awarded the "
-                        + "Pulitzer Prize for his book The Emperor of All " + "Maladies, a “biography” of cancer.") };
-
-        attributeList = Arrays.asList(bookAttr);
-        ArrayList<Attribute> attLis = new ArrayList<>();
-        attLis.addAll(Arrays.asList(bookAttr));
-        attLis.remove(idAttr);
-        attLis.remove(pagesAttr);
-        modifiedAttributeList = (List<Attribute>) attLis;
-
-        bookTuple1 = new ArrayList<>(1);
-        bookTuple1.add(new DataTuple(bookSchema, book1));
-        bookTuple2 = new ArrayList<>(1);
-        bookTuple2.add(new DataTuple(bookSchema, book2));
-
-        dataStoreForOuter = new DataStore(DataConstants.INDEX_DIR + "/join_test_dir_1", bookSchema);
-        dataWriterForOuter = new DataWriter(dataStoreForOuter, analyzer);
-        dataStoreForInner = new DataStore(DataConstants.INDEX_DIR + "/join_test_dir_2", bookSchema);
-        dataWriterForInner = new DataWriter(dataStoreForInner, analyzer);
-        
-        dataWriterForOuter.clearData();
-        dataWriterForInner.clearData();
-    }
-
-    // A helper method to get join result. Called from each test case
-    public List<ITuple> getJoinResults(IOperator outer, IOperator inner, Attribute idAttribute, Attribute joinAttribute,
```

Comment:
This function is moved to the function with the same name in `JoinDistanceHelper.java`.